pennylane/ops/cv.py

```diff
@@ -107,7 +107,7 @@ def num_params(self):
     def _heisenberg_rep(p):
         return _rotation(p[0])
 
-    def adjoint(self, do_queue=False):
+    def adjoint(self, do_queue=True):
```

Comment:
@KetpuntoG how come this change is needed? I ask mostly because the qubit gates all have `do_queue=False` by default!

Reply:
This is for the second problem, otherwise the interferometer would not show anything. In the `default.qubit` all are set to `True` by default. Why are they set to False in CV?

Reply:
Ah, I get it now @KetpuntoG!

What confused me is that the base `Operator` class defines `adjoint=False` as the ‘default’:

https://github.com/PennyLaneAI/pennylane/blob/23b82a4d6b7c5a42d2a4a9e660ac7f61f2919e29/pennylane/operation.py#L717

But the qubit gates implicitly ignore this! For example, for `RX`, we have

https://github.com/PennyLaneAI/pennylane/blob/23b82a4d6b7c5a42d2a4a9e660ac7f61f2919e29/pennylane/ops/qubit/parametric_ops.py#L79-L80

Nice catch 🙂

Reply:
Exactly, in the case of gates, it is implicitly shown  😄